python/feathub/feature_views/transforms/sliding_window_transform.py

```diff
@@ -39,8 +40,13 @@ def __init__(
         :param expr: A FeatHub expression composed of UDF and feature names.
         :param agg_func: The aggregation function or the name of the aggregation
                          function as string such as "MAX", "AVG".
-        :param window_size: The size of the sliding window.
+        :param window_size: The size of the sliding window. If it is timedelta(0), it
+                            means that the sliding window has infinite size. All the
+                            rows are aggregated.
         :param step_size: The step_size specifies how often the sliding windows starts.
+                          If window_size is timedelta(0), step_size can be set to
```

Comment:
It seems a bit redundant to include "If window_size is timedelta(0)" without specifying what happens when window_size != timedelta(0).

How about using the following doc:

The step_size specifies how often the sliding windows starts. If it is timedelta(0), it means that each input will trigger an immediate output of the aggregated result.